crypto/rsa/rsa_impl.c

```diff
@@ -644,11 +635,10 @@ static int mod_exp(BIGNUM *r0, const BIGNUM *I, RSA *rsa, BN_CTX *ctx) {
     goto err;
   }
 
-  /* Turn BN_FLG_CONSTTIME flag on before division operation */
-  pr1 = &local_r1;
-  BN_with_flags(pr1, r1, BN_FLG_CONSTTIME);
+  /* Ensure BN_FLG_CONSTTIME flag is on before division operation */
+  assert(BN_get_flags(r1, BN_FLG_CONSTTIME));
```

Comment:
> No assertion fail, which makes me think that we are reusing old entries in the BN_CTX_POOL that already have the flags set. Maybe we should reset the flags in BN_CTX_get?

Sorry, I didn't see this note. I actually *just* submitted a change to do that to BoringSSL for unrelated reasons: https://boringssl-review.googlesource.com/#/c/7597/.